### ISSUE TITLE
Evaluate/update Sync flow aria-labels for images

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#13816]
See this comment: [https://github.com/mozilla/fxa/pull/13739/files#r931351858](https://github.com/mozilla/fxa/pull/13739/files#r931351858|smart-link) 

A lot of our content-server images are shown via background-image and alt text is provided via `aria-label`. Many of our alt text images used to or currently read the exact same text as the header above it, making the aria-label effectively useless, or not having any read, effectively marking them as decorative images when I don’t believe all of them should be.

We should go through content-server and check for images we’re adding via background-image and decide 1) if they’re decorative or not, in which case removing `aria-label` / `role` is sufficient since screenreaders will skip this, or 2) if alt text should be read and what that alt text should be (might require working with Product), which will require adding `role="presentation"` and `aria-label`s on these images.

Info on decorative images: [https://www.w3.org/WAI/tutorials/images/decorative/](https://www.w3.org/WAI/tutorials/images/decorative/|smart-link)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-5641)

**Comment by cknowles-admin** [comment]
FxA is no longer synchronizing all issues between Jira and Github.  We are closing open issues and will selectively synchronize in the future.
